### PR TITLE
Made extra changes

### PR DESCRIPTION
**Pull request by meshack-mbuvi**

### CODE REVIEW

**Comment by Muriukidavid** [comment]
@meshack-mbuvi please write meaningful commit messsages, I forwarded a link on email abut this

**Comment by meshack-mbuvi** [comment]
got it

File: main.c [hunk 1]
```diff
@@ -38,30 +31,19 @@ int main(void){
...
 		if(data_available()){
-			byte = uart_read();
+			LOOP: byte = uart_read();
+			if (byte>47&&byte<58){//check that data received is an digit
+				blinkLEDS(byte);//call function to blink leds with byte
+					//as the argument
+			}
+			// TODO: check character being given as input to be part of boundary
+			else{
+			goto LOOP;
```
Comment (Muriukidavid) [inline main.c]: rewrite this, remove GOTO commands

File: main.c [hunk 2]
```diff
@@ -38,30 +31,19 @@ int main(void){
 
 	//Loop forever
 	while(1)
-	{
-		toggle_LED1(); 
-		delay();
+	{		
 		if(data_available()){
-			byte = uart_read();
+			LOOP: byte = uart_read();
```
Comment (Muriukidavid) [inline main.c]: stop using labels and goto commands you dont understand :exclamation: :exclamation:

**Pull request closed without merging**